schemas/json/dandi/upload.json

```
@@ -1,7 +1,8 @@
 {
     "properties": {
         "dandiset_id": {
-            "type": "string"
+            "type": "string",
+            "description": "The unique identifier for your dandiset. Will automatically determine whether to upload to the main or staging DANDI server."
```

```suggestion
            "description": "The unique identifier for your dandiset. Will automatically determine whether to upload to the main DANDI archive or the development staging server."
```
Slight rephrase suggestion